### PR TITLE
[FEAT] Add package description to index.rst

### PR DESCRIPTION
**Pull request by MOchiara**

### CODE REVIEW

**Comment by MOchiara** [comment]
This is a draft! Give inputs, it's ugly now I know!

**Comment by MOchiara** [comment]
#78

File: docs/source/index.rst [hunk 1]
```diff
@@ -6,6 +6,23 @@
 Welcome to glidertest's documentation!
 ======================================
 
+Glidertest is a Python package aiming to diagnose possible issues with your glider dataset. The package takes as input data from any glider in OG1 format.
```
Comment (callumrollo) [inline docs/source/index.rst:9]: How about some links like this ` OG1 format <https://github.com/OceanGlidersCommunity/OG-format-user-manual>`_

File: docs/source/index.rst [hunk 2]
```diff
@@ -6,6 +6,23 @@
...
+Glidertest is a Python package aiming to diagnose possible issues with your glider dataset. The package takes as input data from any glider in OG1 format. 
+At the moment, we have implemented functions ranging from basic visualization of your glider data (location, gridding in the dataset, water mass properties, water column structure, etc.) to more complex analysis such as flight model performance when you work with vertical velocities or non-photochemical quenching when you work with optical data. 
+This package serves solely as a diagnostic tool. We occasionally have suggestions for packages or tools that aim to address and solve a particular issue or link to SOP where more processing info can be found.
+
+We recommend consulting best practice guides like the Oxygen SOP (https://oceangliderscommunity.github.io/Oxygen_SOP/README.html) and the other OceanGliders SOPs. We also recommend the GliderTools Python package for processing and to possibly address some common issues with glider data.
+
+We use work from the following papers:
+    * Thomalla et al. 2018 (DOI: https://doi.org/10.1002/lom3.10234)
+    * Frajka-Williams et al. 2011
+    * Bennett (2013)
```
Comment (callumrollo) [inline docs/source/index.rst:18]: Get full citations to these papers!

File: docs/source/index.rst [hunk 3]
```diff
@@ -6,6 +6,23 @@
...
+ 
+We provide an example demo to demonstrate the purpose of the various function and test datasets from SeaExplorer data in the Baltic and Seaglider data in the Labrador Sea.
```
Comment (callumrollo) [inline docs/source/index.rst:20]: "example notebook" maybe? example demo sounds odd

**Review by callumrollo — CHANGES_REQUESTED** [review]
Great additions! I've made a couple of suggestions

**Review by callumrollo — APPROVED** [review]

**Pull request merged**